tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_RTL_POC.py

```diff
@@ -0,0 +1,173 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.settings.app import Settings
+from marionette_driver import By
+
+
+class TestSettingsRTL(GaiaImageCompareTestCase):
+    _page_locator = (By.ID, 'root')
+
+    def setUp(self):
+        GaiaImageCompareTestCase.setUp(self)
+
+    def test_settings_app(self):
+
+        settings = Settings(self.marionette)
+        settings.launch()
+
+        self.take_screenshot('main')
+
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+        self.scroll(self._page_locator, 'down', 600)
+        self.take_screenshot('main')
+
+        settings.open_wifi_settings()
```

Comment:
So in order to extract screenshots from the loop, we have to explicitly mention when to scroll x times.  Then we'll need to put an if statement for each of them as below:
for subpage in ('wifi', 'sim_manager', ...):
   open_method = settings.__getattribute__('open_{}_settings'.format(subpage))
   open_method()
  if subpage == 'wifi':
      self.take_screenshot(subpage)
      self.scroll(subpage)
      self.take_screenshot(subpage)
  if subpage == 'sim_manager':
      self.take_screenshot(subpage)
      self.scroll(subpage)
      self.take_screenshot(subpage)
      self.scroll(subpage)
      self.take_screenshot(subpage)

   settings.return_to_main_menu()

And above is just opening 1 level of submenu.  We also need to open submenus within the submenu (For example, Settings -> Device Information -> More information or Settings -> Accessibility -> Color Filter)  For the person who writes the actual script, wouldn't it be easier to follow the workflow if we don't use loop?